README.md

```diff
@@ -1,13 +1,13 @@
 <p align="center">
-<img height="256" src="https://github.com/iina/iina/raw/master/iina/Assets.xcassets/AppIcon.appiconset/iina-icon-256.png" />
+<img height="256" src="https://github.com/iina/iina/raw/master/iina/Assets.xcassets/AppIcon.appiconset/iina-icon-256.png">
 </p>
 
 <h1 align="center">IINA</h1>
 
 <p align="center">IINA is the <b>modern</b> video player for macOS.</p>
 
-<p align=center>
-<a href="https://iina.io">Website</a> ·
+<p align="center">
+<a href="https://iina.io/">Website</a> ·
```

Comment:
Is there a reason we should prefer these?

Reply:
The trailing slash is not needed in the current version of HTML5, so it can simply be removed ([reference](https://stackoverflow.com/q/7366344/11336855)).

The attribute value is usually enclosed in quotation marks. You can look at lines 1-12 in the same file ([reference](https://www.w3schools.com/html/html_attributes.asp)).

As for the link, in this form it is copied from the address bar of the browser.